tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,22 +6,7 @@
    "source": [
```

Comment:
<p><code></code><code>suggestion</p><p>The basic element needed for your first program is the <code>QuantumCircuit</code>. We begin by creating a&nbsp;<code>QuantumCircuit</code>&nbsp;comprised of three qubits.</p><p></code><code></code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Fixed the backquotes. However, I think the classical bits should be mentioned here.</p>